src/SFA.DAS.ApprenticeFeedback.Jobs/host.json

```diff
@@ -9,7 +9,8 @@
     },
     "logLevel": {
       "default": "None",
-      "Function.SFA.DAS.ApprenticeFeedback.Jobs.Functions.UpdateApprenticeFeedbackTargetTrigger": "Information"
+      "Function.SFA.DAS.ApprenticeFeedback.Jobs.Functions.UpdateApprenticeFeedbackTargetTrigger": "Information",
+      "Function.SFA.DAS.ApprenticeFeedback.Jobs.Functions.ProcessFeedbackTargetVariantsFunction": "Trace"
```

Comment:
I haven't seen any Trace level logging - did you mean to turn it on?

Reply:
changed to information